### DISCUSSION TITLE
國三學生持刀傷害同學，未成年犯無前科註記

### DISCUSSION THREAD

**Discussion by Patsy17** [1101-GS4719-C/Python-Community-GS4719-C#1807] [General]
最近很大的新聞就是國三生遭同學拿刀刺頸!
送醫後救回，但情況不樂觀，恐會成植物人。
學校明明應該是很安全的地方，現在卻發生這種驚人事件...
據知加害人本來就是問題學生，而且未成年犯案並不會有前科註記。

有一些聲音表明應該要區分好、壞學生，不讓他們在同一學校就讀，
這樣霸凌事件、傷人事件可以減少
大家對於這件事情的看法為何？

**Comment by uwje** [comment]
但號外學生通常也會受到環境影響，要怎麼去判斷誰是好學生誰是壞學生便成為一個問題，如果是你的小孩被排到壞學生的學校你會滿意嗎，但感覺的確這樣能杜絕霸凌，但也可能是霸凌狀況更加嚴重，甚至久久出現的砍人事件變得天天上演

**Comment by uwje** [comment]
我認為不是簡單的把好壞學生拆開便能阻止霸凌的發生，畢竟我覺得好壞學生是相對的，在一群人中總是有較好的跟爛的，如何避免好學生變壞跟壞學生繼續沈淪才是重點
